src/store/scrap.js

```diff
@@ -0,0 +1,105 @@
+import { scrapAPI } from 'api';
+import { createAction, handleActions } from 'redux-actions';
+import createRequestSaga, { createRequestSagaActionTypes } from './createRequestSaga';
+import { takeLatest } from 'redux-saga/effects';
+
+const [GETSCRAPLIST, GETSCRAPLIST_SUCCESS, GETSCRAPLIST_FAILURE] = createRequestSagaActionTypes('scrap/GETSCRAPLIST');
+const [DELETESCRAPITEM, DELETESCRAPITEM_SUCCESS, DELETESCRAPITEM_FAILURE] =
+  createRequestSagaActionTypes('scrap/DELETESCRAPITEM');
+const [GETMEMO, GETMEMO_SUCCESS, GETMEMO_FAILURE] = createRequestSagaActionTypes('scrap/GETMEMO');
+const [FIXMEMO, FIXMEMO_SUCCESS, FIXMEMO_FAILURE] = createRequestSagaActionTypes('scrap/FIXMEMO');
+const [WRITEMEMO, WRITEMEMO_SUCCESS, WRITEMEMO_FAILURE] = createRequestSagaActionTypes('scrap/WRITEMEMO');
```

Comment:
```suggestion
const [GET_SCRAP_LIST, GETS_CRAP_LIST_SUCCESS, GET_SCRAP_LIST_FAILURE] = createRequestSagaActionTypes('scrap/GET_SCRAP_LIST');
const [DELETE_SCRAP_ITEM, DELETE_SCRAP_ITEM_SUCCESS, DELETE_SCRAP_ITEM_FAILURE] =
  createRequestSagaActionTypes('scrap/DELETE_SCRAP_ITEM');
const [GET_MEMO, GET_MEMO_SUCCESS, GET_MEMO_FAILURE] = createRequestSagaActionTypes('scrap/GET_MEMO');
const [FIX_MEMO, FIX_MEMO_SUCCESS,FIX_MEMO_FAILURE] = createRequestSagaActionTypes('scrap/FIX_MEMO');
const [WRITE_MEMO, WRITE_MEMO_SUCCESS, WRITE_MEMO_FAILURE] = createRequestSagaActionTypes('scrap/WRITE_MEMO');
```
요렇게 하면 가독성이 좋을 것 같습니다

Reply:
변경 완료 했습니다.